base/src/test/java/uk/ac/ebi/atlas/model/arraydesign/ArrayDesignDaoIT.java

```diff
@@ -38,7 +43,16 @@
     private ArrayDesignDao subject;
 
     private static final String SELECT_IDENTIFIERS_WITH_DESIGNELEMENT_COUNT =
-            "SELECT identifier, COUNT(DISTINCT(designelement)) FROM designelement_mapping GROUP BY identifier";
+            "SELECT identifier, COUNT(DISTINCT(designelement)) AS count FROM designelement_mapping GROUP BY identifier";
+
+    @BeforeAll
+    void beforeAllTests() {
```

Comment:
I'm going to be a bit picky here and say that this method name is a bit repetitive and not very descriptive, if you read it together with the annotation. Something like `populateDatabase` would read a bit better IMO.

Reply:
You’re totally right.